lab-ron/server.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+const http = require('http');
+const cowsay = require('cowsay');
+const parseUrl = require('url').parse;
+const parseQuery = require('querystring').parse;
+const parseBody = require('./lib/parse-json');
+const port = process.argv[2]||3000;
+
+var server = http.createServer(function(req,res){
+  req.url = parseUrl(req.url);
+  var pathname = req.url.pathname;
+  var query = req.url.query = parseQuery(req.url.query);
+
+  if(pathname == '/'){
```

Comment:
Why are you using ==? I would use === unless you have a reason why you want to use ==.